### ISSUE TITLE
All new files have their Language Mode set to COBOL

### ISSUE THREAD

**Issue by pjyoung-ibm** [spgennard/vscode_cobol#337]
**Describe the bug**
When doing `File -> New Text File`, the new file will automatically have the Language Mode set to COBOL. This prevents VSCode from later detecting the probable language of the contents typed/pasted in, and doesn't seem like a sensible default for those creating a new text file.

**To Reproduce**
1. Open a stock VSCode instance with only the COBOL extension installed, and default settings for everything.
2. Create a new text file.
3. See that the file has the COBOL Language Mode selected.

**Expected behavior**
The default Language Mode for a new text file should not be COBOL, as it prevents VSCode from guessing the Language Mode for the contents that end up being typed/pasted in.

**Screenshots**

https://github.com/user-attachments/assets/fd03bd06-d3e3-4f32-a043-f36f81bd1d63

**Desktop (please complete the following information):**
 - OS: macOS 14.5
 - Version: VSCode 1.91.1 | COBOL Extension 10.7.23

**What user & workspace settings are being used?:**
I removed all of my user and workspace settings prior to recreating this issue, so only the defaults for VSCode and this extension.

**Comment by spgennard** [comment]
This is done because a lot of COBOL developers do not use file extensions.

I recommend you use this setting: 

```json
    "files.defaultLanguage": ""
```

**Comment by pjyoung-ibm** [comment]
I disagree that this should be set as the global default for everyone that happens to have this extension installed (who might be working with more than just COBOL files), and believe this should be an opt-in rather than an opt-out. But I guess if this is intentional behavior, then the documentation update is good.

**Comment by spgennard** [comment]
It is a shame vscode has this behaviour, in a ideal world it would check for known languages and then default to this language setting if it fails.

You can also "click" on the assigned language indicator in the editor to change it to a different language.

I will take a further look at the weekend and see if they is anything else can be done.

**Comment by pjyoung-ibm** [comment]
Well, any new file created with "File -> New Text File" (or Ctrl+N/Cmd+N on the keyboard) will have this behavior, which is a common way to create a new file before saving it with a file extension. Yes I can click on the Language Mode to change it, but by default I don't need to do that because VSCode will guess the language mode when I start typing or paste some code in. This is very convenient. Adding a click to always revert from COBOL when I rarely write in COBOL (I do open some COBOL or PL/I files once in a while, which is why I have this installed) is additional work I have to do when creating a new file. Again, this seems like something that should be an opt-in rather than an opt-out. I think it would be better to put this as a recommended change in the README or some other mechanism (perhaps one of the bottom-right pop-up prompts?), rather than have people who don't want this to have to revert a setting that the extension changed. I guess I don't know what percentage of people have this extension installed because they actively do COBOL development everyday, versus what percentage of people just have this installed for the syntax highlighting and other like features that this extension provides. Knowing that would likely help guide which default was "more sensible".

**Comment by spgennard** [comment]
You always opt-in to another extension or just use profiles
